### PR TITLE
ENG-88: Implement configure tab

### PR DESCRIPTION
**Pull request by AfeezGL**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -26,7 +28,7 @@ services:
     ports:
       - 8000:8000
     environment:
-      - DATABASE_URL=postgres://${POSTGRES_USER}:${POSTGRES_PASSWORD}@db:5432/${POSTGRES_DB}
+      - DATABASE_URL=postgres://postgres.ugojraljnvusodkofopv:m9BBOv64a9rtvA2g@aws-0-us-west-1.pooler.supabase.com:6543/postgres
```
Comment (jdamilola) [inline docker-compose.yml:31]: @AfeezGL You shouldn't add your credentials to a public repo.

File: apps/frontend/src/index.css [hunk 2]
```diff
@@ -28,18 +28,6 @@
     }
 }
 
-.tooltip__arrow:before {
```
Comment (kaf-lamed-beyt) [inline apps/frontend/src/index.css:31]: @AfeezGL, this pseudo-element is for the tooltip style in the studio. Did you move it elsewhere?

File: docker-compose.yml (hunk 1 above)
Comment (AfeezGL) [reply docker-compose.yml]: That was a big miss, I'll just roll the credentials.

File: apps/frontend/src/index.css (hunk 2 above)
Comment (AfeezGL) [reply apps/frontend/src/index.css:31]: @kaf-lamed-beyt No, I removed it, radix-ui already provided a `Tooltip.Arrow` primitive, we don't need to create another tooltip arrow. More on that [here](https://www.radix-ui.com/primitives/docs/components/tooltip#arrow)

Comment (kaf-lamed-beyt) [reply apps/frontend/src/index.css:31]: I see. Thanks for spelling that out!

**Pull request merged**